script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -485,112 +493,126 @@ The faint light of the sun dances across my eyelids.
 [sha:2a1910689f710255b374c4ad079a9af24084778e]{
 -- Page 24, Offset 29067.
 -- 兄さんも今朝はそれなりですね。昨夜は夜更かしをしていたようですが」
--- TRANSLATION HERE
+For once, you're quite early yourself, nii-san. Did you not stay up late last night?"
 }
 [sha:7486a56be4d52c1ad871ba4cca9d94e48efe42ae]{
 -- Page 25, Offset 29068.
 -- ああ、それで怒っている風を装っていたのか。
--- TRANSLATION HERE
+Oh, so that's why she was acting all upset.
 }
 [sha:5402a4760aaec2d05301e3f9557a876d4479b502]{
 -- Page 25, Offset 29069.
 -- 家の決まりを守らせる為の威圧。
--- TRANSLATION HERE
+She puts on an intimidating air so that the rules of the house are followed.
 }
 [sha:68199315bad7762fa7748ffd347b5b0f498c1c8d]{
 -- Page 25, Offset 29070.
 -- 少女としてはともかく、家長としては文句のつけようがない。
--- TRANSLATION HERE
+Even if that's behavior unfitting of a maiden,
+ it is the undoubtedly what befits the family head.
```

Comment:
it undoubtedly befits the family head. /

it is undoubtedly befitting of [a/the] family head